jobmaster/cvsJob/cvsJobMaster.go

```diff
@@ -76,7 +78,16 @@ func NewCVSJobMaster(ctx *dcontext.Context, workerID lib.WorkerID, _ lib.MasterI
 	return jm
 }
 
-func (jm *JobMaster) InitImpl(ctx context.Context) error {
+func (jm *JobMaster) Init(ctx context.Context) error {
+	isFirstStartUp, err := jm.BaseMaster.Init(ctx)
```

Comment:
Should `CVSJobMaster` holds an `BaseJobMaster` instead of `BaseMaster` after this PR?